src/clojure/nrepl/middleware/session.clj

```diff
@@ -337,13 +335,18 @@
                            :session session-id})
         (t/send transport (response-for msg :status #{:done}))))))
 
-(defn- close-session
-  "Drops the session associated with the given message."
-  [{:keys [session transport] :as msg}]
+(defn close-session
+  "Drop the given session."
```

Comment:
I know you kept this from the older docstring, but "Close" seems like a better work than "Drop" here.